src/assets/data/defaultConfig.ts

```diff
@@ -122,34 +119,52 @@ export const DefaultNumberOfNotificationsToRescue: number = 12 //
 export const FCMPluginProjectSenderId: string = '430900191220'
 export const DefaultNotificationRefreshTime: number = 900000 // 15 mins in ms
 
-export const DefaultSourceTypeModel: string = 'aRMT-App'
+export const DefaultSourcePrefix = 'aRMT'
+export const DefaultSourceTypeModel: string = `${DefaultSourcePrefix}-App`
 export const DefaultSourceTypeRegistrationBody = {
   sourceTypeCatalogVersion: '1.4.3',
-  sourceTypeModel: 'aRMT-App',
+  sourceTypeModel: DefaultSourceTypeModel,
   sourceTypeProducer: 'RADAR'
   // "deviceTypeId": 1104
 }
 
 export const DefaultEndPoint: string =
   'https://radar-cns-platform.rosalind.kcl.ac.uk/'
-// export const DefaultEndPoint: string = 'https://radar-backend.co.uk/'
-
-export const DefaultSchemaEndpoint =
-  'https://raw.githubusercontent.com/RADAR-base/RADAR-Schemas/add-audio/'
-export const DefaultSchemaSpecExtension = '.yml'
-export const DefaultSchemaSpecEndpoint =
-  DefaultSchemaEndpoint +
-  'specifications/active/aRMT-' +
-  DefaultSourceTypeRegistrationBody.sourceTypeCatalogVersion +
-  DefaultSchemaSpecExtension
-
-export const DefaultProtocolEndPoint: string = DefaultProtocolEndPointExport
 
-export const DefaultSourceProducerAndSecret: string = DefaultSourceProducerAndSecretExport
+// GITHUB SOURCES
+
+export const GIT_API_URI = 'https://api.github.com/repos'
+export const DefaultOrganisation = 'RADAR-Base'
+
+export const DefaultProtocolRepo = 'RADAR-aRMT-protocols'
+// NOTE: Change branch to master when merged
+export const DefaultProtocolBranch = 'test'
```

Comment:
Will you change this value when you build the app for production? It used to be `master`

Reply:
Yup or when we merge before releasing to alpha.